picard/__init__.py

```diff
@@ -38,6 +38,7 @@
 
 
 def version_to_string(version, short=False):
+    """Deprecated: Use picard.version.Version.to_string instead"""
```

Comment:
Should we use https://docs.python.org/3/library/warnings.html#warnings.warn for that?

Reply:
Yes, I think we should do that. Probably with some decorator. We have a few other places to use it